src/exampleTypes.js

```diff
@@ -298,8 +298,15 @@ export default F.stampKey('type', {
     },
     onDispatch(event, extend) {
       // If mutating any group type specific "resetting" keys, set `forceReplaceResponse`
-      let { type, node, previous } = event
+      let { type, node, previous, value } = event
+
       if (type === 'mutate') {
+        // Resetting the drilldown when the node is changed
+        // allows to return expected root results instead of nested drilldown
+        // EX: changing the columns or rows config was not returning the new results
+        if (node.drilldown && !value.drilldown) {
```

Comment:
This looks good, but what if you're mutating to a falsey drilldown? The UI doesn't currently support it, but a falsey drilldown will expand everything: https://github.com/smartprocure/contexture-elasticsearch/blob/master/src/example-types/metricGroups/pivot.js#L99

Reply:
You might be able to fix this by using `_.has`

Reply:
@daedalus28 Ok good eye!  I updated it to what you suggested.